redcap_bridge/project_building.py

```diff
@@ -160,6 +165,15 @@ def combine_values(value1, value2):
     if output_file is not None:
         combined_df.to_csv(output_file, index=False)
 
+    # print number of customizing field
+    print("\nCustomizing", len(customization_df), "fields\n", end='')
+
+    # print the field name
```

Comment:
So this is printing all selected customized fields in one line? Or are they split in multiple lines? @killianrochet Can you provide an example output here?

Maybe a note like: 'The following fields were customized:' would also be nice, no?

Reply:
When the number of fields is less than 100 I display the fields like that : 
Variable / Field Name
ethical_protocol_id
user
...
If the number of fields is superior than 100, I display I list like that:

['ethical_protocol_id' 'user' 'exp_name' 'guid' 'comments_exp'
 'kinarm_recorded_hand' 'kinarm_shoulder' 'kinarm_elbow'
 'eyelink_recorded_eye' 'eyelink_offset_x' 'eyelink_offset_y']

Reply:
Ah, I see. That's a good idea. However printing 100 lines is still a lot, I would adjust the threshold to something smaller, e.g. 10?

Reply:
Yes I think that 100 lines is still unreadable

Reply:
do you mean 'still readable' or 'unreadable'